warehouse/static/sass/blocks/_release.scss

```diff
@@ -101,6 +101,10 @@
     padding-bottom: 0;
     font-size: 20px;
     font-weight: $bold-font-weight;
+
+    .badge {
+      margin-left: 4px;
+    }
```

Comment:
As below, we should instead add a class called `release__version-badge` and add it as `&__version-badge` just above `&__version-date`